### PR TITLE
[WIP] docs: Create project with Che-server's SA on OpenShift with OAuth

### LINKED ISSUE #19717: By default create users namespaces using Che ServiceAccount
### Is your task related to a problem? Please describe.
At this moment when OpenShift OAuth is enabled che-server creating a namespace on behalf of the user's SA.
That brings a lot of manual work for the cluster admins in case if namespaces provisioning is not allowed for the users.
Desired user's permissions in a newly created namespace - TBD

### Describe the solution you'd like
Consider creating user namespaces with che-server's SA even when OpenShift OAuth is enabled

### Describe alternatives you've considered
n/a
### Release Notes Text

By default Che now creates users namespaces using Che ServiceAccount, even if OpenShift OAuth is enabled. This will avoid failures when users don't have enough privileges to create namespaces. An administrator can still change the default and configure Che to create namespaces using users tokens.

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Explain how create project with Che-server's SA on OpenShift with OAuth

### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/19717
Related to https://github.com/eclipse-che/che-server/pull/7

### Specify the version of the product this PR applies to.
7.32

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 1]
```diff
@@ -150,6 +150,32 @@ subjects:
 On {kubernetes}, `che` ServiceAccount must have a cluster-wide `list` and `get` `namespaces` permissions as well as an `admin` role in target namespace.
 ====
 
+[#creating-namespace-with-server-sa]
+== Creating a {orch-namespace} with `che` ServiceAccount
+In situation when users dosn't have enaugh permissions to create a {orch-namespace} and <<pre-creating-namespace>>
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]:
```suggestion
In situation when users dosn't have enough permissions to create a {orch-namespace} and <<pre-creating-namespace>>
```

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 2]
```diff
@@ -150,6 +150,32 @@ subjects:
 On {kubernetes}, `che` ServiceAccount must have a cluster-wide `list` and `get` `namespaces` permissions as well as an `admin` role in target namespace.
 ====
 
+[#creating-namespace-with-server-sa]
+== Creating a {orch-namespace} with `che` ServiceAccount
+In situation when users dosn't have enough permissions to create a {orch-namespace} and <<pre-creating-namespace>>
+is not sutable - the {orch-namespace} can be created with help of `che` ServiceAccount.
```
Comment (themr0c) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]:
```suggestion
is not suitable - the {orch-namespace} can be created with help of `che` ServiceAccount.
```

**Comment by themr0c** [comment]
This PR is introducing following language warnings and suggestions. Can you have a look at them? They are not blockers, but we try to avoid introducing new language warnings.

```
 155:1    warning     Keep sentences short and to     CheDocs.SentenceLength 
                      the point                                              
 155:19   suggestion  Verify your use of 'users'      IBM.Usage              
                      with the word usage                                    
                      guidelines.                                            
 156:44   suggestion  'be created' is passive voice.  IBM.PassiveVoice       
                      Consider rewording in active                           
                      voice.                                                 
 161:145  suggestion  Verify your use of 'that' with  IBM.Usage              
                      the word usage guidelines.                             
 161:158  suggestion  Verify your use of 'allow'      IBM.Usage              
                      with the word usage                                    
                      guidelines.                                            
 161:164  suggestion  Verify your use of 'users'      IBM.Usage              
                      with the word usage                                    
                      guidelines.                                            
 161:180  suggestion  Verify your use of 'new' with   IBM.Usage              
                      the word usage guidelines.                             
 166:23   suggestion  Verify your use of 'using'      IBM.Usage              
                      with the word usage                                    
                      guidelines.                                            
```

**Comment by themr0c** [comment]
I wonder if we should seize the opportunity of some work on this topic to modularize properly the complete story.

* The new content should be in a dedicated procedure file. 
* The story should be decoupled in one assembly + one procedure file per procedure.
* Each procedure should follow the modular docs template. Any "if .. then .. else" is a sign that we should have distinct procedures.

**Comment by skabashnyuk** [comment]
I would like to re-think if this pr is needed or not. I believe that property description might be enough. Please do not progress on this pr.

**Comment by skabashnyuk** [comment]
Property description https://github.com/eclipse-che/che-server/pull/7/files#diff-deca31b9cca47d5b0356dd75fc0fd0c8cb980f3aa029da27b475bdbb37353eb0R553-R555

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 3]
```diff
@@ -150,6 +150,27 @@ subjects:
 On {kubernetes}, `che` ServiceAccount must have a cluster-wide `list` and `get` `namespaces` permissions as well as an `admin` role in target namespace.
 ====
 
+[#disabling-creating-namespace-with-server-sa]
```
Comment (sparkoo) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:152]: WDYT about negating this? I mean write somewhere on this page that namespaces are created by `che-server` service account and rewrite this paragraph into "How to create namespaces with user account"?

Comment (skabashnyuk) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc:152]: You right. Originally this article was build when it was disabled by default. After it was changed to true I don't see a value in it anymore.

**Pull request closed without merging**